{
  ProjectWorkspace workspace=TestDataHelper.createProjectWorkspaceForScenario(this,"test_call_flow",temporaryFolder);
  workspace.setUp();
  ProcessResult suiteTestResult=workspace.runBuckCommand("test","//:example");
  suiteTestResult.assertExitCode("Test should fail because @AfterClass method is supposed to call exit(42)",1);
  assertThat(suiteTestResult.getStderr(),containsString("exit code 42"));
}
